helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -472,11 +472,15 @@ public void updateDeploymentGroupHosts(final String name, final List<String> hos
           }
         }
 
+        log.info("starting zookeeper transaction for updateDeploymentGroupHosts on "
```

Comment:
But if the transaction failed, an error message will be logged right next to this one, right? Or is it the word "starting" that is an issue? In the end all I really wanted to do was log the operations about to be undertaken.